BankSDK/GiniBankSDKExample/GiniBankSDKExample/AppDelegate.swift

```diff
@@ -8,10 +8,12 @@
 import UIKit
 
 @UIApplicationMain
-final class AppDelegate: UIResponder, UIApplicationDelegate {
-
-var coordinator: AppCoordinator!
-var window: UIWindow?
+    final class AppDelegate: UIResponder, UIApplicationDelegate {
+    private let appsheme = "BankSDKExtension://"
```

Comment:
[Request] Typo and also other variable names use camel case so
```suggestion
    private let appScheme = "BankSDKExtension://"
```